src/ExperimentalInternetComputer.mo

```diff
@@ -27,7 +27,8 @@ module {
     let pre = Prim.performanceCounter(0);
     comp();
     let post = Prim.performanceCounter(0);
-    post - pre
+    /// The `performance_counter` API costs 200 extra instructions, we subtract 200 to reflect the real computation cost of `comp`.
```

Comment:
Maybe it's cleaner to measure the cost of doing nothing (i.e. two calls to the counter but with nothing in between) to get the overhead?  This way it'll be correct even if the replica changes.,